### PR TITLE
Updates to use Mozilla redash docker image.

### LINKED ISSUE #108: Switch to Mozilla redash and update links
Since we are targeting STMO we should use that within out ```docker-compose.yml``` and also update any links referencing ```getredash/redash```

### PR DESCRIPTION
**Pull request by jrbenny35**
Fixes #108 

This switches redash-ui-tests to using mozilla's fork of Redash. Also updated a link in the docs to point to the Mozilla Redash wiki page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,6 @@ Check out who has already contributed to **redash-ui-tests**
 [first]: https://github.com/mozilla/redash-ui-tests/labels/good%20first%20issue
 [git]: https://git-scm.com/
 [redash help]: https://redash.io/help/
-[redash]: https://github.com/getredash/redash
+[redash]: https://wiki.mozilla.org/Telemetry/Custom_dashboards_with_re:dash
```
Comment (hackebrot) [inline README.md]: Hey @jezdez @rafrombrc what do you see as the home of the stmo project? The wiki page, the GitHub fork, the running stmo instance? 🤔

File: pages/queries.py [hunk 2]
```diff
@@ -66,7 +66,8 @@ def click_dropdown_menu(self, text=None) -> typing.Any:
         for item in (i for i in items if i.text in text):
             if text == "Fork":
                 item.click()
-                return QueryDetailPage(self.selenium, self.base_url)
+                page = QueryDetailPage(self.selenium, self.base_url)
+                return page.wait_for_page_to_load()
```
Comment (hackebrot) [inline pages/queries.py]: Is there a difference in UI behavior between upstream and fork?

File: README.md (hunk 1 above)
Comment (jezdez) [reply README.md]: This is more up-to-date: https://docs.telemetry.mozilla.org/tools/stmo.html

File: pages/queries.py (hunk 2 above)
Comment (jrbenny35) [reply pages/queries.py]: No, I just forgot to call the ```wait_for_page_to_load()``` and noticed it was causing some intermittent failures

Comment (hackebrot) [reply pages/queries.py]: Okay, great! 👍

**Pull request merged**